### PR TITLE
Fixed Mac build error for #97 & updated to auto-create Julia.app.

### PR DESCRIPTION
**Pull request by NHDaly**
Adds a commit to this branch to fix the mac build error in #97:

The current master (before PR #97) downloads a hand-modified version of
electron v0.35.2, hosted on AWS. This version has been modified so that
when it runs, the process is named Julia, the icons are Julia's, and the
menu bar name is Julia.

Before this commit (at 20dfac1), #97 was failing because it downloaded
stock Electron.app, rather than the hand-modified version.

This commit adds logic to install.jl that programatically applies the
same hand-modifications to the downloaded Electron.app:
 - rename the app to Julia.app
 - update its icons.
 - And update its plist to reflect the new name.
Finally, this commit tells process.jl where to look for the newly
renamed binary. (I chose to also rename the binary just to make the
find/replace in the .plist simpler.)

### CODE REVIEW

File: src/AtomShell/install.jl [hunk 1]
```diff
@@ -18,6 +22,11 @@ function install()
       download("https://github.com/electron/electron/releases/download/v$version/$file")
       run(`unzip -q $file`)
       rm(file)
+      run(`mv Electron.app Julia.app`)
+      run(`mv Julia.app/Contents/MacOS/Electron Julia.app/Contents/MacOS/Julia`)
+      run(`sed -i .bak 's/Electron/Julia/' Julia.app/Contents/Info.plist`)
```
[the remark this answers is not in the document]
Comment (pfitzseb) [reply src/AtomShell/install.jl]: Hm, I don't *think* this should cause problems, but I also have absolutely zero experience with macs :)

**Comment by pfitzseb** [comment]
Nice sleuthing, and thanks for the patch! :)

I'll probably just merge this once CI is green for the latest commit and we'll see how it goes.

File: src/AtomShell/install.jl (hunk 1 above)
[the remark this answers is not in the document]
Comment (NHDaly) [reply src/AtomShell/install.jl]: haha okay sounds good to me. Enjoy your brand new iMac! 😄

**Comment by NHDaly** [comment]
:] Thanks! It was fun! I'm glad to help. I'll be following along!

**Pull request merged**